docs/2.0/docs/pipelines/tutorials/deploying-to-aws-gov-cloud.md

```diff
@@ -84,15 +80,17 @@ This section covers the Pipelines configuration required to deploy an AWS S3 buc
 
 3. We'll now use that `vars.yaml` file as input to [boilerplate](https://github.com/gruntwork-io/boilerplate) to generate the Terragrunt code for the OIDC Provider and IAM roles.  From the root of your repository, run the following command:
 
+<!-- TODO: Update the version of the architecture catalog templates below -->
+
 <Tabs>
 <TabItem value="GitHub" label="GitHub">
 ```bash
-boilerplate --template-url "git@github.com:gruntwork-io/terraform-aws-architecture-catalog.git//templates/github-actions-single-account-setup?ref=main" --output-folder . --var-file vars.yaml --non-interactive
+boilerplate --template-url "git@github.com:gruntwork-io/terraform-aws-architecture-catalog.git//templates/github-actions-single-account-setup?ref=X.Y.Z" --output-folder . --var-file vars.yaml --non-interactive
```

Comment:
Just checking: is there a ticket to remind us to do this? Can we reference it in the TODO above so we know it's done and can get rid of the TODO?